src/main/java/biomesoplenty/api/item/BOPItems.java

```diff
@@ -99,6 +99,13 @@
     public static Item flower_basket;
     public static Item jar_empty;
     public static Item jar_filled;
+    
+    /*
+     *  Use net.minecraftforge.fluids.UniversalBucket.getFilledBucket(
+     *          net.minecraftforge.common.ForgeModContainer.getInstance().universalBucket,
+     *          biomesoplenty.common.fluids.QuicksandFluid.instance
+     *      )
+     */
     public static Item sand_bucket;
     public static Item honey_bucket;
     public static Item blood_bucket;
```

Comment:
You  might as well just remove these since they'll no longer be in use anyways. And then probably the comment along with them, since it'll no longer make sense just randomly sitting there.